x-pack/filebeat/module/fortinet/firewall/ingest/event.json

```diff
@@ -0,0 +1,518 @@
+        "field": "fortinet.firewall.srcport",
+        "target_field": "source.port",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.locport",
+        "target_field": "source.port",
+        "ignore_missing": true,
+        "if": "ctx.source?.port == null"
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.user",
+        "target_field": "source.user.name",
+        "ignore_missing": true,
+        "if": "ctx.fortinet?.firewall?.user != 'N/A'"
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.saddr",
+        "target_field": "source.address",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.agent",
+        "target_field": "user_agent.original",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.file",
+        "target_field": "file.name",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.filesize",
+        "target_field": "file.size",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.level",
+        "target_field": "log.level",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.logid",
+        "target_field": "event.code",
+        "ignore_missing": true,
+        "if": "ctx.event?.code == null"
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.msg",
+        "target_field": "message",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.policyid",
+        "target_field": "rule.id",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.proto",
+        "target_field": "network.iana_number",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.dir",
+        "target_field": "network.direction",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.direction",
+        "target_field": "network.direction",
+        "ignore_missing": true,
+        "if": "ctx.network?.direction == null"
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.service",
+        "target_field": "network.protocol",
+        "ignore_missing": true
+      }
+    },
+    {
+      "lowercase": {
+        "field": "network.protocol",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.error_num",
+        "target_field": "error.code",
+        "ignore_missing": true
+      }
+    },
+    {
+      "rename": {
+        "field": "fortinet.firewall.hostname",
+        "target_field": "url.domain",
```

Comment:
Another interesting mapping that could really use a comment

Reply:
Next commit will come in YAML, added a JSON commit first so its easier to see the changes I did between the newest JSON and the earlier